src/index.ts

```diff
@@ -1749,6 +1749,8 @@ function FlatpickrInstance(
       self._input.classList.add("active");
       triggerEvent("onOpen");
       positionCalendar(positionElement);
+      self.calendarContainer.focus();
```

Comment:
Without this line, `focusOnDay()` doesn't do anything for reasons I'm unsure of.